### PR TITLE
Add Logmatic.io provider.

### PR DESCRIPTION
**Pull request by JulienBreux**
Add Logmatic.io provider.

❤️ Monolog

### CODE REVIEW

File: src/Monolog/Formatter/LogmaticFormatter.php [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+     * @param string
+     */
+    protected $appname = '';
+
+    /**
+     * Set hostname
+     *
+     * @param string $hostname
+     */
+    public function setHostname(String $hostname) {
```
Comment (stof) [inline src/Monolog/Formatter/LogmaticFormatter.php]: the scalar typehint is ``string``, not ``String``

File: src/Monolog/Handler/LogmaticHandler.php [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+use Monolog\Logger;
+use Monolog\Formatter\FormatterInterface;
+use Monolog\Formatter\LogmaticFormatter;
+
+/**
+ * @author Julien Breux <julien.breux@gmail.com>
+ */
+class LogmaticHandler extends SocketHandler
+{
+    const LOGMATIC_URI = '/v1/';
```
Comment (stof) [inline src/Monolog/Handler/LogmaticHandler.php]: no need for this constant IMO (and its name is bad anyway, as this is not a URI)

**Comment by gpolaert** [comment]
@JulienBreux It's very helpful. Thanks!

**Comment by Seldaek** [comment]
Thanks

**Pull request merged**